homeassistant/components/mqtt/client.py

```diff
@@ -924,9 +924,11 @@ def _async_mqtt_on_connect(
         if birth:
 
             async def publish_birth_message(birth_message: PublishMessage) -> None:
```

Comment:
May be change the functions name ? and add a docst?

Reply:
adjusted in 26e53dc8fc460b79d0b88fa19c8f28dc1698a569